compiler/rustc_metadata/src/rmeta/encoder.rs

```diff
@@ -900,14 +900,13 @@ fn should_encode_mir(tcx: TyCtxt<'_>, def_id: LocalDefId) -> (bool, bool) {
             let always_encode_mir = tcx.sess.opts.unstable_opts.always_encode_mir;
             (is_const_fn, needs_inline || always_encode_mir)
         }
-        // Closures can't be const fn.
         DefKind::Closure => {
```

Comment:
Should we simply merge this arm with the previous one?

Reply:
Oh yeah, I guess the only diff is that call to `is_const_default_method`